conf/gunicorn/development-websocket.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+
+bind = "localhost:5001"
+timeout = 0
+graceful_timeout = 0
+worker_class = "h.websocket.Worker"
+proc_name = "websocket"
+reload = True
+
+
+if "H_GUNICORN_CERTFILE" in os.environ:
+    certfile = os.environ["H_GUNICORN_CERTFILE"]
+
+
+if "H_GUNICORN_KEYFILE" in os.environ:
+    keyfile = os.environ["H_GUNICORN_KEYFILE"]
```

Comment:
This duplicates the `H_GUNICORN_CERTFILE` and `H_GUNICORN_KEYFILE` from `development-app.py` but it's only four lines and it's something that hardly ever changes. I don't think it's worth introducing a shared file for especially since importing will be a fiddle